modules/bitgo/src/v2/coins/hbar.ts

```diff
@@ -249,6 +249,7 @@ export class Hbar extends BaseCoin {
         amount: txJson.amount.toString(),
         address: txJson.to,
         memo,
+        tokenId: txJson.tokenId,
```

Comment:
this is inside `explainTransaction`, do I need tokenName instead of tokenId as well?